mono/tests/Makefile.am

```diff
@@ -1223,14 +1223,22 @@ CI_PR_DISABLED_TESTS = \
 
 # appdomain-threadpool-unload.exe creates 100 appdomains, takes too long with llvm
 LLVM_DISABLED_TESTS = \
-	finally_block_ending_in_dead_bb.exe \
 	appdomain-threadpool-unload.exe
 
+# https://github.com/mono/mono/issues/8319
+LLVM_DISABLED_TESTS = \
+	finally_block_ending_in_dead_bb.exe \
+	generic-stack-traces2.2.exe
+
 # https://github.com/mono/mono/issues/6490
 LLVM_DISABLED_TESTS += \
 	weak-fields.exe
 
-LLVM = $(filter --llvm, $(MONO_ENV_OPTIONS))
+if ENABLE_LLVM
```

Comment:
does ENABLE_LLVM here mean that llvm is actually used or just that it is available?

Reply:
That it is available. Unfortunately we have no good way of checking if we have are running with llvm enabled.

Reply:
Since it doesn't seem to work, and the tests have only started to fail recently (https://jenkins.mono-project.com/job/test-mono-mainline-linux-fullaot+llvm/507/) I don't think we should disable them, and just have somebody look at what the regression was.